File: Idyl_Icaro/GerenciadorVenda.cs
class GerenciadorVenda
{
    private GerenciadorCliente _clientes = new GerenciadorCliente();
    public void novaVenda(Cliente cliente, int idVendedor, string data){
        this._clientes.inserir(cliente);
        // print idVendedor
        // print data
    } 
}

File: Nathan/GerenciadorVenda.java
public class GerenciadorVenda
{
    public void newSale(Object cliente, int idVendedor, string data){
        new GerenciadorCliente().insertClient(cliente);
    } 

     public void  removeCliente(Object cliente, int idVendedor, string data){
        new GerenciadorCliente().removeCliente(cliente);
    } 
}